src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/ZipFilesSummaryRepository.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.entity.reports;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+public interface ZipFilesSummaryRepository extends JpaRepository<Envelope, UUID> {
+
+    @Query(
+        nativeQuery = true,
+        value = "SELECT env.jurisdiction, env.status, e1.container, "
+            + "e1.zipfilename, e1.createdDate, e2.completedDate FROM \n"
+            + "(SELECT container, zipfilename, MIN(createdat) AS createdDate "
+            + "FROM process_events  \n"
+            + "WHERE CAST(createdat AS DATE) = :date\n"
```

Comment:
the other SQL report uses `date(createdat)`. Worth checking which one is appropriate and keep it consistent imho

Reply:
ah `DATE`, i knew i saw it somewhere

Reply:
yeah I'll change the query to use`date(createdat)`

Reply:
Updated the query to use `date(createdat)`